packages/block-editor/src/components/block-navigation/block.js

```diff
@@ -46,20 +46,26 @@ export default function BlockNavigationBlock( {
 	const [ isHovered, setIsHovered ] = useState( false );
 	const [ isFocused, setIsFocused ] = useState( false );
 	const { clientId } = block;
-	const isDragging = useSelect(
+	const { isDragging, getParents } = useSelect(
 		( select ) => {
-			const { isBlockBeingDragged, isAncestorBeingDragged } = select(
-				'core/block-editor'
-			);
+			const {
+				isBlockBeingDragged,
+				isAncestorBeingDragged,
+				getBlockParents,
+			} = select( 'core/block-editor' );
 
-			return (
-				isBlockBeingDragged( clientId ) ||
-				isAncestorBeingDragged( clientId )
-			);
+			return {
+				isDragging:
+					isBlockBeingDragged( clientId ) ||
+					isAncestorBeingDragged( clientId ),
+				getParents: getBlockParents,
```

Comment:
Couldn't we just extract `blockParents` here?

```diff
-				getParents: getBlockParents,
+				blockParents: getBlockParents( clientId ),
```

Reply:
Done!